File: JS/TwoSum.js
let arr=[2,8,5,4,7,6]
let target=9
let found=false
for(let i=0; i<arr.length; i++)
{
    let diff=target-arr[i]
    for(let j=i+1; j<arr.length; j++)
    {
         if(arr[j]==diff)
    {
        found=true
        console.log(`found ${arr[i]} and ${arr[j]} at ${i} ${j}`)
        break
    }
    
    }
    if(found)
        break
}

File: JAVA/TwoSum.java
class TwoSum{
  public static void main(String[] args) {
       //System.out.println("Hello");
       int[] arr={2,8,5,7,11};
       int target=9;
       for(int i=0; i<arr.length; i++)
       {
        int diff=target-arr[i];
       
      for(int j=i+1; j<arr.length; j++)
      {
        if(diff==arr[j])
          System.out.println(arr[j]+" "+arr[i]+" "+"at "+i+" "+j);
       }
      }    
  }
}